TriblerGUI/widgets/trustpage/js/legend.js

```diff
@@ -0,0 +1,90 @@
+
+
+function drawLegend(data) {
+    var linkSize = d3.scaleLinear().domain([data.min_transmission, data.max_transmission]).range([config.link.strokeWidthMin, config.link.strokeWidthMax]);
```

Comment:
Please add a space between `drawLegend` and `(data)`, in line with our JS code style until now;
and you could consider breaking line 4 into two pieces